### ISSUE TITLE
Last update broke everything

### ISSUE THREAD

**Issue by Kiborgik** [nextauthjs/next-auth#8078]
### Environment

It tell something about migration to v5 but it still v4
and give a link to https://nextjs.authjs.dev/v5 but this web site doesn't work

### Reproduction URL

no

### Describe the issue

error: export{};

### How to reproduce

update to last

### Expected behavior

working

**Comment by ndpniraj** [comment]
I am currently using 4.22.2 and it is the lates I guess and it has lots of bug from NextAuth is not function to all the above errors that others are facing. I don't know if this is a bug or docs are not updated.

**Comment by insik-han** [comment]
next-auth@4.22.2

```ts
import { } from 'next-auth/next/index // It works
import { } from 'next-auth/next'; It dose not work
```

But we'd rather downgrade to next-auth@4.21.1 or wait for a fixed version than use the latest version😇

**Comment by thevuong** [comment]
There are so many things changing!

![Screenshot 2023-07-18 at 10 33 31 PM](https://github.com/nextauthjs/next-auth/assets/13298232/f8bdc5ae-9fc3-4328-bd9a-cfbb5ad28cb2)

**Comment by Jni75** [comment]
Revert to 4.22.1 fix the issue

<img width="582" alt="image" src="https://github.com/nextauthjs/next-auth/assets/12665174/d2686f9d-1ae8-4cfe-a760-272045b8306f">

**Comment by c0nf1den71al** [comment]
> Revert to 4.22.1 fix the issue
> 
> <img alt="image" width="582" src="https://user-images.githubusercontent.com/12665174/254319718-d2686f9d-1ae8-4cfe-a760-272045b8306f.png">

I can confirm downgrading does fix temporarily:

package.json
```
    "next-auth": "4.22.1",
```

**Comment by thevuong** [comment]
Upgrade to v5

https://authjs.dev/guides/upgrade-to-v5

**Comment by c0nf1den71al** [comment]
> Upgrade to v5
> 
> https://auth-docs-git-feat-nextjs-auth-authjs.vercel.app/guides/upgrade-to-v5

@thevuong No, this will not work as the latest version (4.22.2) is the version which does not work.

**Comment by lilonghe** [comment]
Yesterday morning, `pnpm i next-auth`  ->  "next-auth": "0.0.0-pr.6777.c5550344"  
When i find this question, the latest is `4.22.2`, update two hours ago, and it's also doesn't work too

Hope the v5 will work when I wake up, now i need to go to sleep.

**Comment by chrxmvtik** [comment]
Recently I have updated from 4.19.2 to both 4.22.1 and 4.22.2 - latest versions are causing this problem:

```[next-auth][error][CLIENT_FETCH_ERROR] 
https://next-auth.js.org/errors#client_fetch_error undefined {
  error: {},
  url: 'http://localhost:3000/api/auth/session',
  message: undefined
}```

Tried multiple ideas, modifying env variables, adding some sort of configuration - but still no luck, simply next-auth won't work for me.

Reverting back to 4.19.2 fixed this problem.

**Comment by JosePascual** [comment]
> 4.22.1

Sabe here.

**Comment by bcanfield** [comment]
I got the following error when I went from `4.22.1` to `4.22.2`: 

`next-auth/jwt is deprecated. if you are not ready to migrate, keep using next-auth@4`

Reverting to 4.22.1 fixed it.

**Comment by ItaiAxelrad** [comment]
Getting errors even after rolling back to 4.22.1

<img width="638" alt="Screenshot 2023-07-18 at 1 25 10 PM" src="https://github.com/nextauthjs/next-auth/assets/51974602/dbe91e11-4e38-4fa7-8961-2c431b1e8221">

**Comment by yohannlog** [comment]
I have reverted to 4.22.1 but I keep an error in my docker build:

`Failed to compile.

./pages/api/auth/[...nextauth].ts:149:16
Type error: This expression is not callable.
  Type 'typeof import("/app/node_modules/next-auth/next")' has no call signatures.

  147 | }
  148 | 
> 149 | export default NextAuth(authOptions);
      |                ^
  150 | `
  
  Any idea ?

**Comment by realokd** [comment]
I was facing the same issue while implementing  next-auth in my personal project. Shifted back to @4.22.1 and everything work fine

**Comment by balazsorban44** [comment]
Hi, could anyone add a reproduction repository?

**Comment by balazsorban44** [comment]
Hi, This is fixed in `4.22.3`, please upgrade! The issue was that some gitignored leftover files made it into the published package folder. :facepalm:

**Comment by renardeinside** [comment]
I have this problem again in:
```
"next-auth": "^4.24.7",
```
